app/models/preference.rb

```diff
@@ -45,7 +45,7 @@ class Preference < Ekylibre::Record::Base
                             decimal language integer record
                             spatial_reference_system string], predicates: true
   @@conversions = { float: :decimal, true_class: :boolean, false_class: :boolean, fixnum: :integer }
-  cattr_reader :reference
+  attr_reader :reference
```

Comment:
`@@reference` is a class variable, why did you change this to be `attr_reader` instead of the previously `cattr_reader`?